src/installation/index.md

```diff
@@ -1,160 +1,201 @@
-[see Usage section of the `espup` Readme]: https://github.com/esp-rs/espup#usage
+You can also directly download pre-compiled [release binaries] or use [cargo-binstall].
+
 [release binaries]: https://github.com/esp-rs/espup/releases
 [cargo-binstall]: https://github.com/cargo-bins/cargo-binstall
 
-### Other installation methods
+Once `espup` is installed, do the following:
+
+1. Install all the necessary tools to develop Rust applications for all supported Espressif targets by running:
+    ```sh
+    espup install
+    ```
+
+    `espup` will create an export file that contains some environment variables required to build projects:
 
-- Using [esp-rs/rust-build] installation scripts. This was the recommended way in the past, but now the installation scripts are feature frozen and all new features will only be included in `espup`. See the repository README for instructions.
-- Building the Rust compiler with `Xtensa` support from source. This process is computationally expensive and can take one or more hours to complete depending on your system. It is not recommended unless there is a major reason to go for this approach. See instructions in the [Installing from Source section of the esp-rs/rust repository].
+    - Unix systems - `$HOME/export-esp.sh`
+    - Windows - `%USERPROFILE%\export-esp.ps1`
 
-[esp-rs/rust-build]: https://github.com/esp-rs/rust-build
-[Installing from Source section of the esp-rs/rust repository]: https://github.com/esp-rs/rust#installing-from-source
+2. On Unix systems, make sure to source this file in every terminal before building any application: `. $HOME/export-esp.sh`
+    > On Windows systems, no need to source the file. It is only created to show the modified environment variables.
 
-## Rust with `std` runtime
 
-Regardless of the target architecture, if you want to build a project using the [`std` approach], you will also need:
-- [ESP-IDF]: Espressif IoT Development Framework as it's used as our hosted environment.
-  - This is handled by [`esp-idf-sys`] (a crate that all `std` projects need to use) by default. See [ESP-IDF installation note] for details on how ESP-IDF can be installed.
-- [`ldproxy`] crate:  Simple tool to forward linker arguments given to [`ldproxy`] to the actual linker executable. The crate can be found in the [esp-rs/embuild] repository. To install it, use the following command:
-  - `cargo install ldproxy`
+After running `espup install`:
 
+- `no_std` (bare-metal) applications should work out of the box
+- `std` applications require additional software covered in [`std` Development Requirements][rust-esp-book-std-requirements]
+
+### What espup Installs
+
+To enable support for Xtensa targets, `espup` installs the following tools:
+
+- Espressif Rust fork with support for `Xtensa` targets
+- `nightly` toolchain with support for `RISC-V` targets
+- `LLVM` [fork][llvm-github-fork] with support for `Xtensa` targets
+- [GCC toolchain][gcc-toolchain-github-fork] as it is used as linker
```

Comment:
I would suggest using `- [GCC toolchain][gcc-toolchain-github-fork] that links the final binary`. For `no_std` projects, we also use GCC as a linker (that is the reason `espup` downloads it, otherwise we will rely on `esp-idf-sys` )  and in, this case, it does not compiles the ESP-IDF.

Reply:
Ah I see, I didn't know that no_std non-esp-idf-sys projects also used gcc for linking.

Reply:
It is something temporary, but yes. In the future, we plant to use `lld` in `no_std`, see https://github.com/esp-rs/esp-hal/pull/357.

Please, @f-hollow do you mind changing the line as I suggested?
```sugestion
- [GCC toolchain][gcc-toolchain-github-fork] that links the final binary
```

Reply:
Updated the line to

> - [GCC toolchain][gcc-toolchain-github-fork] that links the final binary